src/main/java/racingCar/Car.java

```diff
@@ -0,0 +1,36 @@
+package racingCar;
+
+public class Car {
+	private String name;
+	private int position;
+
+	public Car(String name) {
+		validateName(name);
+		this.name = name;
+		this.position = 0;
+	}
+
+	private void validateName(String name) {
+		if (name.length() > 5)
+			throw new IllegalArgumentException("이름은 5자 이하여야 합니다.");
+	}
+
+	public void move(int number) {
+		if(number>3){
```

Comment:
```suggestion
		if (number > 3) {
```
적절한 공백을 활용하여 가독성을 높일수 있을것 같아요 😄